### PR TITLE
CNV-20469: Provide UI to manage subscription for RHEL VMs - part 1

### PR DESCRIPTION
**Pull request by avivtur**
## 📝 Description

PR includes:
- New section to Overview > Settings > Cluster to manage the subscription for RHEL machines
- Rearrange sections under Overview > Settings > Cluster.

[Jira issue](https://issues.redhat.com//browse/CNV-20469)
[UX doc](https://docs.google.com/document/d/1nn6POVp8xNrzlz12SH70k6oGf4-umAFq26SWjxJSu20/edit)
## 🎥 Demo

Screenshot - 2023-07-10T123110 771
Screenshot - 2023-07-10T121743 019
org-id-popover
activation-key-popover

### CODE REVIEW

**Comment by metalice** [comment]
i think the loading of installed, chanel, update will never stop loading is subscription isnt exist.. please check it :)

File: src/views/clusteroverview/SettingsTab/ClusterTab/components/AutomaticSubscriptionRHELGuests/components/AutomaticSubscriptionForm/AutomaticSubscriptionForm.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      setOrganizationID(subscriptionData?.organizationID);
+    }
+  }, [activationKey, loaded, organizationID, subscriptionData]);
+
+  if (!loaded) return <Skeleton />;
+
+  const isDisabled =
+    !isAdmin ||
+    isEqualObject(subscriptionData, { activationKey, organizationID }) ||
+    (!isEmpty(organizationID) && isEmpty(activationKey)) ||
```
Comment (metalice) [inline src/views/clusteroverview/SettingsTab/ClusterTab/components/AutomaticSubscriptionRHELGuests/components/AutomaticSubscriptionForm/AutomaticSubscriptionForm.tsx:44]: what about a case of both are empty?
simply  || (isEmpty(organizationID)|| isEmpty(activationKey))

File: src/views/clusteroverview/SettingsTab/ClusterTab/components/AutomaticSubscriptionRHELGuests/components/AutomaticSubscriptionForm/AutomaticSubscriptionForm.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    (isEmpty(organizationID) && !isEmpty(activationKey));
+
+  const handleSubmit = () => {
+    if (!isDisabled) {
```
Comment (metalice) [inline src/views/clusteroverview/SettingsTab/ClusterTab/components/AutomaticSubscriptionRHELGuests/components/AutomaticSubscriptionForm/AutomaticSubscriptionForm.tsx:48]: !isDisabled && updateSubscription(activationKey, organizationID)

File: src/utils/hooks/useSubscriptionData/useSubscriptionData.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+  const updateSubscription = async (activationKey: string, organizationID: string) => {
+    setLoading(true);
+    const updatedConfigMap = produce(featureConfigMap, (draftCM) => {
+      if (isEmpty(draftCM?.data)) draftCM.data = {};
+      draftCM.data[AUTOMATIC_SUBSCRIPTION_ACTIVATION_KEY] = activationKey;
+      draftCM.data[AUTOMATIC_SUBSCRIPTION_ORGANIZATION_ID] = organizationID;
+    });
+
+    k8sUpdate({
```
Comment (metalice) [inline src/utils/hooks/useSubscriptionData/useSubscriptionData.ts:41]: its async function u can use await instead of then

**Comment by avivtur** [comment]
> i think the loading of installed, chanel, update will never stop loading is subscription isnt exist.. please check it :)

On dev clusters we need to have DS branding to avoid see this properly, it's not a bug :)

**Comment by metalice** [comment]
> > i think the loading of installed, chanel, update will never stop loading is subscription isnt exist.. please check it :)
> 
> On dev clusters we need to have DS branding to see this properly, it's not a bug :)

great, just make sure there isnt any loading issue on ds as i remember its looking for subscription

**Pull request merged**